src/oncall/scheduler/daily-round-robin.py

```diff
@@ -0,0 +1,442 @@
+from datetime import datetime, timedelta
```

Comment:
This should inherit from either the default or round-robin scheduler. Probably round-robin, as that's what you're trying to do here. Too much copy-paste here.

In addition, I don't think this does round-robin scheduling, since this is copied from the default scheduler (which schedules to maximize time from previous shifts).